### PR TITLE
[docs] Add regression test for weekday function

### PR DESCRIPTION
**Pull request by smallhibiscus**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Add regression test for weekday function .

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: regression-test/suites/query/sql_functions/datetime_functions/test_date_function.groovy [hunk 1]
```diff
@@ -21,7 +21,7 @@ suite("test_date_function", "query") {
     def tableName = "test_date_function"
 
     sql """ SET enable_vectorized_engine = TRUE; """
-
+    sql """ DROP TABLE IF EXISTS ${tableName} """
     sql """
             CREATE TABLE ${tableName} (
```
Comment (Gabriel39) [inline regression-test/suites/query/sql_functions/datetime_functions/test_date_function.groovy:26]: CREATE TABLE IF NOT EXISTS

**Review by Gabriel39 — CHANGES_REQUESTED** [review]

File: regression-test/suites/query/sql_functions/datetime_functions/test_date_function.groovy (hunk 1 above)
Comment (smallhibiscus) [reply regression-test/suites/query/sql_functions/datetime_functions/test_date_function.groovy]: done

**Review by Gabriel39 — APPROVED** [review]
LGTM

**Review by yiguolei — COMMENTED** [review]
lgtm

**Review by yiguolei — APPROVED** [review]
lgtm

**Pull request merged**